packages/apollo-server-cache-redis/src/RedisClusterCache.ts

```diff
@@ -10,7 +10,15 @@ export class RedisClusterCache extends BaseRedisCache {
 
   constructor(nodes: ClusterNode[], options?: ClusterOptions) {
     const clusterClient = new Redis.Cluster(nodes, options)
-    super(clusterClient);
+    super({
+      del: clusterClient.del.bind(clusterClient),
+      flushdb: clusterClient.flushdb.bind(clusterClient),
+      mget(...keys: Array<string>): Promise<Array<string | null>> {
+        return Promise.all(keys.map(key => clusterClient.get(key).catch(() => null)))
```

Comment:
I'm raising a pretty big eyebrow at that "ignore all errors" catch, but I see that it's copied from the previous version, so not gonna worry about it.